### PR TITLE
Fixes statement arguments and Driver name

### PR DESCRIPTION
**Pull request by fedefernandez**
After some tests in a real app, I've found some issues. This PR fixes these issues:
- The driver name should be the real class name
- The statement arguments can't be empty. The previous implementation always has at least one element.
- The method `toStringArray` should return values and never `null` values (if a `null` exists it returns the "NULL" string)
- Removes unnecessary plugin from `core` module

Please @juanpedromoreno could you review? Thanks

### CODE REVIEW

File: core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ class PreparedStatementArguments extends StatementArguments {
   private[this] var argumentsList: NonEmptyList[mutable.Map[Int, AnyRef]] = NonEmptyList(mutable.Map.empty)
 
   // Prepared statements count from 1
```
Comment (juanpedromoreno) [inline core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala:21]: The `var` is now 0, should this comment updated as well?

File: core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala [hunk 2]
```diff
@@ -33,14 +33,14 @@ class PreparedStatementArguments extends StatementArguments {
...
-    maxIndex = 1
+    maxIndex = 0
   }
 
   override def toStringArray: Array[String] = toArray map { value =>
     Option(value) match {
       case Some(v: String) => v
       case Some(v) => v.toString
-      case None => javaNull
+      case None => "NULL"
```
Comment (juanpedromoreno) [inline core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala]: I'm wondering if this constant should be pulled up in a kind of `javaNullString` val.

**Comment by juanpedromoreno** [comment]
Minor comments but code looks good! :+1:

File: core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala (hunk 1 above)
Comment (fedefernandez) [inline core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala:21]: The comment is fine, the prepared statements count from 1. 0 means no arguments.

File: core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala (hunk 2 above)
Comment (fedefernandez) [inline core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala]: :+1:

**Pull request merged**